rainier-core/src/main/scala/com/stripe/rainier/core/SBCModel.scala

```diff
@@ -0,0 +1,48 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+import com.stripe.rainier.sampler._
+
+trait SBCModel[L, T] {
+  implicit val rng: RNG = ScalaRNG(1528673302081L)
+  def sbc: SBC[L, T]
+  def main(args: Array[String]): Unit = sbc.animate(HMC(1), 10000, 1000)
```

Comment:
we probably want the implementations of this trait to be able to override these sampling params

Reply:
👍